src/Directory.Build.props

```diff
@@ -51,6 +48,8 @@
     <CopyVsixManifestToOutput>false</CopyVsixManifestToOutput>
     
     <NoWarn>$(NoWarn);NU5125</NoWarn>
+
+    <DiscoverEditorConfigFiles>true</DiscoverEditorConfigFiles>
```

Comment:
Why isn't this on by default? Are users expected to opt into this to avoid editor configs from being ignored?

Reply:
It isn't turned on yet. It will be turned on by default for 16.2 GA.